src/unix/pty.cc

```diff
@@ -203,7 +203,11 @@ NAN_METHOD(PtyFork) {
 
   // termios
   struct termios* term = new termios();
+#if defined(IUTF8)
   term->c_iflag = ICRNL | IXON | IXANY | IMAXBEL | BRKINT | IUTF8;
+#else
```

Comment:
Shouldn't we keep the lack of support option in https://github.com/Tyriar/node-pty/pull/42/commits/ce42cb6e38311247b3797bc9bd4d485d7505e932?

```
#else
# error No UTF-8 is possible.
```